From 5b15a69c0bc5741fc35c49db387ff26828ce2a26 Mon Sep 17 00:00:00 2001
From: Markus Schweikhardt <markus.schweikhardt@intel.com>
Date: Thu, 5 Nov 2015 15:44:33 +0100
Subject: [PATCH 028/102] ASoC: Intel: Boards: New machine driver integrated
 for GP MRB board

  Intial version of the Bxt GP MRB machine driver added.
  The machine driver is able to handle the four channel amplifier
  FE playback stream and the stereo FE capture stream of the AM/FM
  tuner.

Signed-off-by: Markus Schweikhardt <markus.schweikhardt@intel.com>

Conflicts:
	sound/soc/intel/boards/Makefile
	sound/soc/intel/boards/bxt_gpmrb.c

Conflicts:
	sound/soc/intel/Kconfig
---
 sound/soc/intel/boards/Kconfig  | 9 +++++++++
 sound/soc/intel/boards/Makefile | 2 ++
 2 files changed, 11 insertions(+)

diff --git a/sound/soc/intel/boards/Kconfig b/sound/soc/intel/boards/Kconfig
index 73aa94c..4c5559f 100644
--- a/sound/soc/intel/boards/Kconfig
+++ b/sound/soc/intel/boards/Kconfig
@@ -403,6 +403,15 @@ config SND_SOC_INTEL_CNL_RT274_MACH
 	   Say Y or m if you have such a device. This is a recommended option.
 	   If unsure select "N".
 
+config SND_SOC_INTEL_BXT_GPMRB_MACH
+	tristate "ASoC Audio driver for BXT IVI GP MRB"
+	select SND_SOC_INTEL_SST
+	default n
+	help
+	   This adds support for ASoC machine driver for Broxton IVI GP MRB platform
+	   Say Y if you have such a device
+	   If unsure select "N".
+
 config SND_SOC_INTEL_APLI_LHCRB_DUMMY_MACH
 	tristate "ASoC Audio driver for APLI LHCRB Dummy Codec"
 	depends on X86_INTEL_LPSS && I2C
diff --git a/sound/soc/intel/boards/Makefile b/sound/soc/intel/boards/Makefile
index 9adc437..8c25cba 100644
--- a/sound/soc/intel/boards/Makefile
+++ b/sound/soc/intel/boards/Makefile
@@ -35,6 +35,7 @@ snd-soc-apli_lhcrb_wm8731-objs := apli_lhcrb_wm8731.o
 snd-soc-apli_lhcrb_aic3107M-objs := apli_lhcrb_aic3107M.o
 snd-soc-apli_lhcrb_aic3107S-objs := apli_lhcrb_aic3107S.o
 snd-soc-apli_lhcrb_generic-objs := apli_lhcrb_generic.o
+snd-soc-bxt_gpmrb-objs := bxt_gpmrb.o
 
 obj-$(CONFIG_SND_SOC_INTEL_HASWELL_MACH) += snd-soc-sst-haswell.o
 obj-$(CONFIG_SND_SOC_INTEL_BYT_RT5640_MACH) += snd-soc-sst-byt-rt5640-mach.o
@@ -72,3 +73,4 @@ obj-$(CONFIG_SND_SOC_INTEL_APLI_LHCRB_WM8731_MACH) += snd-soc-apli_lhcrb_wm8731.
 obj-$(CONFIG_SND_SOC_INTEL_APLI_LHCRB_AIC3107_MACH) += snd-soc-apli_lhcrb_aic3107M.o
 obj-$(CONFIG_SND_SOC_INTEL_APLI_LHCRB_AIC3107_MACH) += snd-soc-apli_lhcrb_aic3107S.o
 obj-$(CONFIG_SND_SOC_INTEL_APLI_LHCRB_GENERIC_MACH) += snd-soc-apli_lhcrb_generic.o
+obj-$(CONFIG_SND_SOC_INTEL_BXT_GPMRB_MACH) += snd-soc-bxt_gpmrb.o
-- 
1.9.1

